docker/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM phusion/baseimage:0.10.1
+LABEL maintainer "chevdor@gmail.com"
+
+RUN apt-get update && \
+	apt-get upgrade -y && \
+	apt-get install -y cmake pkg-config libssl-dev && \
+	curl https://sh.rustup.rs -sSf | sh && \
```

Comment:
i get the following error when it tries to run the `curl` command on macOS:
```bash
info: downloading installer
rustup: Unable to run interactively. Run with -y to accept defaults, --help for additional options
The command '/bin/sh -c curl https://sh.rustup.rs -sSf | sh' returned a non-zero code: 1
```

Reply:
i fixed the error by changing it to `curl https://sh.rustup.rs -sSf | sh -s -- -y` (i.e. appending ` -s -- -y`), as recommended in this thread https://github.com/rust-lang-nursery/rustup.rs/issues/1031